_docs/12-component-controllers.md

```diff
@@ -33,4 +33,48 @@ But sometimes, you do need an object with a longer lifecycle. *Component control
 
 There is a only a one-way communication channel between the component and its component controller - you can only pass data off of a component to a component controller. A component has no reference its corresponding component controller. This is by design. 
 
-To pass data from a component to its controller, expose a `@property` on the component in a class extension. The controller can read those properties in `didUpdateComponent`.
+To pass data from a component to its controller, expose a `@property` on the component in a class extension. The controller can initialize itself with the properties here in `initWithComponent:`. If these properties will be changing in subsequent state changes (i.e. a new component is being created with different values for these properties), keep them up to date in `didUpdateComponent`.
```

Comment:
> The controller can initialize itself with the properties here in `initWithComponent:`.

Should that sentence be written a bit differently? Maybe "the controller can initialize itself with the properties in `initWithComponent:`" instead?